### ISSUE TITLE
Build of 6.1.0-post branch and v6.1.0 tag fails

### ISSUE THREAD

**Issue by mikebin** [confluentinc/common#317]
Building the 6.1.0-post branch of v6.1.0 tag of this repo fails with:

```
[ERROR] Failed to execute goal io.confluent:resolver-maven-plugin:0.5.0:resolve-kafka-range (default) on project common-parent: : MojoExecutionException: Could not find any ccs-kafka artifacts. -> [Help 1]
```

Issue appears to be with Maven resolver plugin, which is looking for `6.1.0` artifacts instead of `6.1.0-ccs`. The following is a workaround:

```
mvn install -Dconfluent.version.range=6.1.0-ccs
```

However, this should ideally be fixed in `pom.xml`.

**Comment by andrewegel** [comment]
Yup - Internally discussed, I think adding `-Dskip.maven.resolver.plugin=true` is a more viable work around until we can address this in 6.1.1

**Comment by mikebin** [comment]
Editing the original summary to avoid confusion on the workaround.

**Comment by fuhuoyeyou** [comment]
In the master branch, the mvn install is still failed with this parameter.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file (custom-install) on project common-parent: The specified file '/Users/the_path/common/installed_pom.xml' not exists -> [Help 1]

```

**Comment by tg2018** [comment]
6.1.1-post 
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file (custom-install) on project common-parent: The specified file '/Users/the_path/common/installed_pom.xml' not exists -> [Help 1]

**Comment by samizzy** [comment]
@fuhuoyeyou @tg2018 I resolved this by setting `<custom.install.phase>` to `none`
